library/common/config/config.cc

```diff
@@ -70,6 +89,34 @@ const std::string config_header = R"(
 - &trust_chain_verification VERIFY_TRUST_CHAIN
 - &virtual_clusters []
 
+!ignore local_error_defs: &local_error_config
+  "@type": type.googleapis.com/envoymobile.extensions.filters.http.local_error.LocalError
+
+!ignore network_defs: &network_configuration_config
+  "@type": type.googleapis.com/envoymobile.extensions.filters.http.network_configuration.NetworkConfiguration
+  enable_drain_post_dns_refresh: *enable_drain_post_dns_refresh
+  enable_interface_binding: *enable_interface_binding
+
+!ignore dfp_defs: &dfp_config
+  "@type": type.googleapis.com/envoy.extensions.filters.http.dynamic_forward_proxy.v3.FilterConfig
+  dns_cache_config: &dns_cache_config
+    name: base_dns_cache
+    preresolve_hostnames: *dns_preresolve_hostnames
+    dns_lookup_family: *dns_lookup_family
+    host_ttl: 86400s
+    dns_min_refresh_rate: *dns_min_refresh_rate
+    dns_refresh_rate: *dns_refresh_rate
+    dns_failure_refresh_rate:
+      base_interval: *dns_fail_base_interval
+      max_interval: *dns_fail_max_interval
+    dns_query_timeout: *dns_query_timeout
+    typed_dns_resolver_config:
+      name: *dns_resolver_name
+      typed_config: *dns_resolver_config
```

Comment:
I like the cleanup here, but any of these that contain aliases need to be in `config_template` string rather than `config_header` string in order to maintain their ability to be overridden via EngineBuilders. Should be sufficient to simply relocate this block.